fedramp-integrated-inventory-workbook/deployment/inventory/handler.py

```diff
@@ -0,0 +1,26 @@
+# (c) 2019 Amazon Web Services, Inc. or its affiliates. All Rights Reserved.
+# License:
+# This sample code is made available under the MIT-0 license. See the LICENSE file.
+from .readers import AwsConfigInventoryReader
+from .reports import CreateReportCommandHandler, DeliverReportCommandHandler
+
+def lambda_handler(event, context):
+    inventory = AwsConfigInventoryReader(lambda_context=context).get_resources_from_all_accounts()
+
+    report_path = CreateReportCommandHandler().execute(inventory)
+    report_url = DeliverReportCommandHandler().execute(report_path)
+
+    return {'statusCode': 200,
+            'body': {
+                    'report': { 'url': report_url }
+                }
+            }
+
+if __name__ == "__main__":
+    class Context(object):
+        def __init__(self):
+            self.invoked_function_arn = "arn:aws-us-gov:lambda:us-east-1:123456789012:function:testing"
```

Comment:
Any idea what this is here for? Looks like a mock

Reply:
Initially thought it was used for testing, but upon further inspection it doesn't seem to do anything at the moment. Removing for now.